src/Resources/contao/templates/modules/mod_breadcrumb.html5

```diff
@@ -2,14 +2,163 @@
 
 <?php $this->block('content'); ?>
 
-  <ul itemscope itemtype="http://schema.org/BreadcrumbList">
-    <?php foreach ($this->items as $item): ?>
-      <?php if ($item['isActive']): ?>
-        <li class="active<?php if ($item['class']): ?> <?= $item['class'] ?><?php endif; ?> last" itemscope itemtype="http://schema.org/ListItem" itemprop="itemListElement"><span itemprop="name"><?= $item['link'] ?></span></li>
+  <ol itemscope="" itemtype="http://schema.org/BreadcrumbList">
+    <?php $path_number = 0; foreach ($this->items as $item): ?>
+      <?php if ( $path_number == 0 ): ?>
+        <?php if ($item['isActive']): ?>
+          <li class="active'<?php if ($item['class']): ?> <?= $item['class'] ?><?php endif; ?> last"><?= $item['link'] ?></li>
+        <?php else: ?>
+          <li<?php if ($item['class']): ?> class="<?= $item['class'] ?>"<?php endif; ?>><a href="<?= $item['href'] ?>" title="<?= $item['title'] ?>"><?= $item['link'] ?></a></li>
+        <?php endif; ?>
       <?php else: ?>
-        <li<?php if ($item['class']): ?> class="<?= $item['class'] ?>"<?php endif; ?> itemscope itemtype="http://schema.org/ListItem" itemprop="itemListElement"><a href="<?= $item['href'] ?>" title="<?= $item['title'] ?>" itemprop="url"><span itemprop="name"><?= $item['link'] ?></span></a></li>
+        <?php if ($item['isActive']): ?>
+          <li itemprop="itemListElement" itemscope="" itemtype="http://schema.org/ListItem" class="active<?php if ($item['class']): ?> <?= $item['class'] ?><?php endif; ?> last"><span itemprop="item" itemscope="" itemtype="http://schema.org/Thing"><span itemprop="name"><?= $item['link'] ?></span><link itemprop="url" href="<?= $item['href'] ?>" /></span><meta itemprop="position" content="<?= $path_number ?>" /></li>
```

Comment:
How will browsers render the `<meta>` tag?

Reply:
They don't render it at all. `<meta>` tags are allowed in the `<body>` in HTML5. A lot of examples on http://schema.org take advantage of that.